kod/object/active/holder/room.kod

```diff
@@ -1790,6 +1790,20 @@ messages:
       return;
    }
 
+   RestartRoomBackgroundMusic()
```

Comment:
Maybe "ResendRoomBackgroundMusic" is a better name?  There's nothing in the code here about restarting; it's up to the client to decide what it wants to do with this message.

Reply:
Done.